file-formats/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -26,19 +26,18 @@ INTERFACE zif_aff_cota_v1
       "! Communication type
       "! $default {@link zif_aff_cota_v1.data:co_comm_type.http}
       communication_type          TYPE ty_comm_type,
-      "! <p class="shorttext">Connection Target Class</p>
-      "! Connection target class
-      "! $required
+      "! <p class="shorttext">Communication Target Class</p>
+      "! Communication target class
       connection_target_class     TYPE zif_aff_types_v1=>ty_object_name_30,
```

Comment:
I see, you changed the terminology from "connection" to "communication", which is fine for me. Are you still able to rename `connection_target_class` to `communication_target_class`. This would be more consistent, wouldn't it?